src/js/select2/i18n/ko.js

```diff
@@ -0,0 +1,44 @@
+define(function () {
+  // English
+  return {
+    errorLoading: function () {
+      return '결과를 불러올 수 없습니다.';
+    },
+    inputTooLong: function (args) {
+      var overChars = args.input.length - args.maximum;
+
+      var message = '너무 깁니다. ' + overChars + ' 글자 지워주세요.';
+
+      if (overChars != 1) {
+        message += 's';
+      }
+
+      return message;
+    },
+    inputTooShort: function (args) {
+      var remainingChars = args.minimum - args.input.length;
+
+      var message = '너무 짧습니다. ' + remainingChars + ' 글자 더 입력해주세요.';
+
+      return message;
+    },
+    loadingMore: function () {
+      return '불러오는 중…';
+    },
+    maximumSelected: function (args) {
+      var message = '최대 ' + args.maximum + '개까지만 선택 가능합니다.';
+
+      if (args.maximum != 1) {
```

Comment:
I don't think this is needed? It doesn't look like there are any advanced pluralization rules to worry about.